### ISSUE TITLE
Anbox uses a lot of power, makes device hot

### ISSUE THREAD

**Issue by cyberpunkedu** [ubports/anbox#10]
Device: FP2
Channel: 16.04/devel
Anbox Version: 2018-09-16 whatever is most recent.

When running anbox apps, the device gets quite hot. It seems to be requiring a lot of power. App performance is a bit laggy, especially on touch input. Not sure if it's related or not.

**Comment by mariogrip** [comment]
does it only when the app is visible or does the same happen when the app is not showing? and does this happen of the device is locked?

**Comment by cyberpunkedu** [comment]
The testing is still a bit inconclusive. 
I can say that it's definitely warmer only when running anbox applications in the foreground but it seems to only be the case with the wifi on, so maybe it's a wifi kernel thing?  Battery life also seems worse with the anbox kernel.

Anyway, maybe need more testing and other input. Results are a bit inconsistent so far.

**Comment by mihaelmilea** [comment]
My anbox keyboard is also laggy and battery drainage is constant since I installed anbox (turbo):
![screenshot20180919_234317482](https://user-images.githubusercontent.com/30057613/45861535-bf1d6c80-bd75-11e8-90f8-b364bdb4aab7.png)

**Comment by mihaelmilea** [comment]
`anbox-tool disable` saves battery life. I enable it when I need anbox.

**Comment by ITwrx** [comment]
Yes, it seems to be more inefficient than i would expect it to need to be. Hopefully there is a large virtualization/graphics gain to be had somewhere, b/c right now the performance is probably not going to be good enough to actually use day to day, at least for the apps i want to use. Currently, CSipSimple and Jami. i'm using a Nexus 5 with UBports 16.04 stable.
